tsdb/block.go

```diff
@@ -61,11 +61,8 @@ type IndexReader interface {
 	// beyond the lifetime of the index reader.
 	Symbols() index.StringIter
 
-	// SortedLabelValues returns sorted possible label values.
-	SortedLabelValues(name string) ([]string, error)
-
-	// LabelValues returns possible label values which may not be sorted.
-	LabelValues(name string) ([]string, error)
+	// LabelValues returns sorted possible label values.
+	LabelValues(name string, sortedValues bool) ([]string, error)
```

Comment:
To be strict, can we switch parameter to be first? It will be then consistent with :hugs:  https://github.com/prometheus/prometheus/blob/24b897164f5bc5178d399f34843edd72967fb667/tsdb/querier.go#L122